src/Filter/ModelAutocompleteFilter.php

```diff
@@ -86,10 +86,6 @@ protected function handleMultiple(ProxyQueryInterface $query, string $alias, Fil
 
     protected function handleModel(ProxyQueryInterface $query, string $alias, FilterData $data): void
     {
-        if (empty($data->getValue())) {
```

Comment:
maybe this should check for hasValue?

Reply:
Already done in the filter method, 

If we want to add check if would be aroung falsy value like `''`, `null` or `[]` but I don't know which one are really possible to land here.